projects/cdk/schematics/utils/angular-json-manipulations.ts

```diff
@@ -33,57 +34,73 @@ export function addStylesToAngularJson(
     )} to angular.json manually.`;
 
     return updateWorkspace(workspace => {
-        const project = getProject(options, workspace);
+        const projects = getProjects(options, workspace);
 
-        if (!project) {
+        if (!projects.length) {
             context.logger.warn(
                 `[WARNING]: Target project not found. ${MANUAL_MIGRATION_TIPS}`,
             );
             return;
         }
 
-        let targetOptions;
+        for (let project of projects) {
+            let targetOptions;
 
-        try {
-            targetOptions = getProjectTargetOptions(project, 'build');
-        } catch {
-            context.logger.warn(
-                `[WARNING]: No buildable project was found. ${MANUAL_MIGRATION_TIPS}`,
-            );
-            return;
-        }
+            try {
+                targetOptions = getProjectTargetOptions(project, 'build');
+            } catch {
+                context.logger.warn(
+                    `[WARNING]: No buildable project was found. ${MANUAL_MIGRATION_TIPS}`,
```

Comment:
The same

https://github.com/Tinkoff/taiga-ui/pull/3144#discussion_r1035798129